src/main/java/com/scmp/framework/context/FrameworkConfigs.java

```diff
@@ -134,4 +134,10 @@ public FrameworkConfigs(ConfigurableApplicationContext context, Environment env)
 
 	@Value("${NOTIFICATION_FAILED_CASE_EXCLUDE_LIST:#{''}}")
 	private String failedCaseExcludeList;
+
+	@Value("${NOTIFICATION_FAILED_CASE_SLACK_USERID:#{''}}")
+	private String failedCaseNotificationSlackUserId;
```

Comment:
@benliscmp can we use `GITLAB_USER_LOGIN` if `NOTIFICATION_FAILED_CASE_SLACK_USERID` is empty ?
NOTIFICATION_FAILED_CASE_SLACK_USERID -> GITLAB_USER_LOGIN -> ``

Reply:
Updated to @Value("${NOTIFICATION_FAILED_CASE_SLACK_USERID:${GITLAB_USER_LOGIN:}}")